ccb_vaultic/ccb_vaultic.c

```diff
@@ -0,0 +1,1021 @@
+/*
+ * ccb_vaultic.c
+ *
+ * Copyright (C) 2023 wolfSSL Inc.
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfSSL is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
+ */
+
+/* System includes */
+#include <stdlib.h>  /* For NULL */
+#include <string.h>  /* For memset/cpy */
+
+/* wolfSSL configuration */
+#include "wolfssl/options.h"
+
+/* wolfCrypt includes */
+#include "wolfssl/wolfcrypt/cryptocb.h"
+#include "wolfssl/wolfcrypt/error-crypt.h"
+
+#include "wolfssl/wolfcrypt/wc_port.h"
+#include "wolfssl/wolfcrypt/types.h"
+
+#include "wolfssl/wolfcrypt/hash.h"  /* For HASH_FLAGS and types */
+#include "wolfssl/wolfcrypt/rsa.h"   /* For RSA_MAX_SIZE */
+
+/* Local include */
+#include "ccb_vaultic.h"
+
+#ifndef HAVE_CCBVAULTIC
+/* Provide dummy implementations of callbacks */
+int ccbVaultIc_Init(ccbVaultIc_Context *c)
+{
+    return CRYPTOCB_UNAVAILABLE;
+}
+
+/* Close the Wisekey VaultIC library. */
+void ccbVaultIc_Cleanup(ccbVaultIc_Context *c) {
```

Comment:
For consistency please have the function brace on new line. Thanks

Reply:
I'll correct that.  I gotta get code formatting rules put in...